CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 1.4.0
  * Added new date types for x-axis(Quarter, Hours, Minutes, Seconds)
+ * Added ability to use custom date format for tooltip dates
```

Comment:
1.4.0 has already been released at [GitHub](https://github.com/Microsoft/powerbi-visuals-gantt/releases/tag/1.4.0)
Please increase version of the visual.